### ISSUE TITLE
Apparent contradictions between documentation and code

### ISSUE THREAD

**Issue by ndarilek** [hoodiehq-archive/hoodie-plugin-global-share#14]
This has been perplexing me. From the README:

If you want every user to be able to edit each user's entries in your application, this plugin is the right one for you. If you just want to publish content with read-only rights for users, you should choose [public shares](https://github.com/hoodiehq/hoodie-plugin-shares). 

From the code:

```
  if (!hasRole(oldDoc.createdBy)) {
    throw {
      unauthorized: 'Only creator can edit this'
    };
  }
```

The code seems to imply that only creators can edit, but the README contradicts this.

Am I missing something?

**Comment by gr2m** [comment]
cc @caolan

**Comment by steve21124** [comment]
yes, I thought other user can edit too but the code not allow it. :)
